apps/portal/src/components/questions/typeahead/RoleTypeahead.tsx

```diff
@@ -1,31 +1,25 @@
 import { useState } from 'react';
 
-import { JobTitleLabels } from '~/components/shared/JobTitles';
+import { useJobTitleOptions } from '~/components/shared/JobTitlesTypeahead';
```

Comment:
We should probably split up the hooks into separate files so that this component doesn't need to import the entire `JobTitlesTypeahead` module (though this is a non-issue with tree shaking enabled). Still better to split up :)